docs/Getting Started/Debian/Debian Bullseye Root on ZFS.rst

```diff
@@ -93,6 +93,11 @@ entered at the console. Performance is good, but LUKS sits underneath ZFS, so
 if multiple disks (mirror or raidz topologies) are used, the data has to be
 encrypted once per disk.
 
+Per-user encrypted homdirs encrypts only the datasets for each user. The system
```

Comment:
typo in "homdirs", but also expand it.
"Per-user encrypted home directories encrypts only the home directory datasets for each user."